frontend/packages/console-app/src/components/quick-starts/QuickStartDrawer.tsx

```diff
@@ -78,10 +80,12 @@ const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({
               size="xl"
               style={{ marginRight: 'var(--pf-global--spacer--md)' }}
             >
-              {quickStart?.spec.displayName}
-            </Title>
-            <Title headingLevel="h6" size="md" className="text-secondary">
-              {`${quickStart?.spec.duration} minutes`}
+              {quickStart?.spec.displayName}{' '}
+              <span className="co-quick-start-drawer__duration">
+                <Text component={TextVariants.small} className="text-secondary">
+                  {`${quickStart?.spec.duration} minutes`}
+                </Text>
+              </span>
```

Comment:
Why are you using `Text` ? Fairly certain `Text` is only useful inside of `TextContent`.

```suggestion
              <small className="co-quick-start-drawer__duration text-secondary">
                  {`${quickStart?.spec.duration} minutes`}
              </span>
```

Reply:
updated

Reply:
I was using Text as I simply moved a pre-existing block inside the Header without modifying it